test/fuzz/transformation_equation_instruction_test.cpp

```diff
@@ -13,6 +13,7 @@
 // limitations under the License.
 
 #include "source/fuzz/transformation_equation_instruction.h"
+
```

Comment:
I typically don't leave a new line because we're not including a header corresponding to the .cpp file, exactly (as this is a test.cpp file).  But I'm not sure what best practice is here.

Reply:
Fixed in a6ad6b3.